android/app/src/main/AndroidManifest.xml

```diff
@@ -33,8 +35,8 @@
             android:largeHeap="true">
       <meta-data android:name="firebase_analytics_collection_deactivated" android:value="true" />
       <meta-data
-              android:name="com.google.firebase.messaging.default_notification_icon"
-              android:resource="@drawable/ic_stat_status_notification" />
+        android:name="com.google.firebase.messaging.default_notification_icon"
+        android:resource="@drawable/ic_stat_ic_notification" />
```

Comment:
I'm getting a build error regarding this resource:
```
 AAPT: No resource found that matches the given name (at 'resource' with value '@drawable/ic_stat_ic_notification').
```

Is this change intentional?